### ISSUE TITLE
Login token error

### ISSUE THREAD

**Issue by ziziz** [jotta/jotta-cli-issues#91]
jottad version    : 0.6.18443
jotta-cli version : 0.6.18443

I cant login with the login-token from your website. Im using Windows 10 and have tried both in CMD and PowerShell.

I have tried multiple times and ensured that only the token is passed to the command line.

```
PS .\jotta-cli.exe login
```
```
accept license (yes/no): yes
```
```
Enter your personal login token. You can generate a personal login token 
by visiting the Security tab under account settings. This is typically found at: 
https://www.jottacloud.com/web/secure

Personal login token:<TOKEN FROM WEBSITE>
```
```
Logging in. Please wait...
ERROR
Sorry, the data you pasted was not parseable.
Please copy-paste the entire contents of the token that was generated and nothing else.

The parsing error returned was: unexpected end of JSON input

```
Expected:
Successful login.

Are you supposed to enter the token in JSON format?

**Comment by Kimbsen** [comment]
I have identified the issue and it seems to be that cmd.exe is chopping a few of the characters of the string you are pasting. It seems to be limited to 254 chars and our string can exceed this some times. 

Until we work around this you can use for instance git bash ( https://git-scm.com/downloads ) to run jotta-cli login as it does not have this problem.

Sorry about that. It should have been caught in testing. I'll get it fixed asap.

**Comment by magnuhho** [comment]
Create a text file containing:
yes
[token]
[empty line]

Then, in cmd:
type <textfile> | "C:\Program Files (x86)\JottaCli\cli\jotta-cli.exe" login

**Comment by Kimbsen** [comment]
That was an interesting work around. Not exactly sure why that works, but it does :)

In other news, I have found a workaround for the normal login flow and will probably be releasing it later today.

**Comment by magnuhho** [comment]
Using pipe with the exe file makes the content of the text file stdin, so it's basically just the text file typing instead of the keyboard :-)

**Comment by Kimbsen** [comment]
Indeed, but why is that buffer not being limited to 254 chars while it is when manually typing/pasting it. 🤷‍♂

**Comment by Kimbsen** [comment]
released 0.6.18626 which fixes this issue.
